libc/unistd_fsdir.c

```diff
@@ -46,8 +45,9 @@ TEST_SETUP(unistd_fsdir)
 	memset(buf, 0, sizeof(buf));
 
 	/* clear/create file */
-	fd = fopen(filename, "w");
-	fclose(fd);
+	filep = fopen(filename, "w");
```

Comment:
@damianloew @niewim19 
IMHO the filename should be immutable (`define`?).

To run tests on host we need to parametrize the cwd (we can't create files in `/` on host without superuser). Also it would be beneficial to somehow run the same test on different filesystems (so with different cwd).

Please change file-related variable names `fd` should be used for **integer file descriptor file**, for `FILE*` (**FILE pointer**) you may use `fp` or `filep` or similar